packages/gui/src/components/plot/add/PlotAddNumberOfPlots.tsx

```diff
@@ -309,6 +309,17 @@ export default function PlotAddNumberOfPlots(props: Props) {
               />
             </FormControl>
           </Grid>
+          <Grid xs={12} item>
+            <FormControl variant="filled" fullWidth>
+              <TextField
+                name="plotNFTContractAddr"
+                type="text"
+                variant="filled"
+                placeholder="Plot NFT Plot Target Address"
```

Comment:
This should use the `t` macro for localization. You'll need to add `t` as an import from @lingui/macro, i.e.:

`import { Trans, t } from '@lingui/macro';`

```suggestion
                placeholder={t`Plot NFT Plot Target Address`}
```

Reply:
Fixed, thanks!